### PR TITLE
RedisInsight -  1.13.0 release notes, Proxy guide, new environment variables

### PR DESCRIPTION
**Pull request by GnaneshKunal**
@nermiller Please review it

Also, 1.13.0 should be release before merging the PR. I'll mention it here.

### CODE REVIEW

**Comment by nermiller** [comment]
No problem, @GnaneshKunal.

File: content/ri/using-redisinsight/proxy.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+     proxy_read_timeout     900;
+     proxy_set_header       Host $host;
+ }
+}
+```
+
+![RIPROXYPATH](/images/ri/riproxypath.png)
+
+
+#### RIPROXYPREFIX - Dynamic Subpath
```
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:133]:
```suggestion
### RIPROXYPREFIX dynamic subpath
```

File: content/ri/using-redisinsight/proxy.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+
+When `RIPROXYPREFIX` is being set to a value, default being `X-Forwarded-Prefix`, RedisInsight extracts the path from this field in the HTTP Header. So the subpath is actually set by the Proxy server and not RedisInsight and this variable just tells which HTTP header field to check for the proxy subpath. Using this approach, multiple proxies can send requests to same RedisInsight with different proxy subpaths.
```
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:136]:
```suggestion
When `RIPROXYPREFIX` is being set to a value, the default being `X-Forwarded-Prefix`, RedisInsight extracts the path from this field in the HTTP Header. Therefore, the subpath is actually set by the Proxy server and not RedisInsight. This variable just tells which HTTP header field to check for the proxy subpath. Using this approach, multiple proxies can send requests to same RedisInsight with different proxy subpaths.
```

File: content/ri/using-redisinsight/proxy.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+
+{{< note >}}
+
+When Dynamic Subpath is used, RedisInsight is reachable both from the normal path, i.e., `localhost:8001` and the one from Dynamic subpath.
```
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:140]:
```suggestion
When the dynamic subpath is used, RedisInsight is reachable both from the normal path, i.e., `localhost:8001` and the one from dynamic subpath.
```

File: content/ri/using-redisinsight/proxy.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+{{< /note >}}
+
+
+##### Example - Multiple Nginx to same RedisInsight
```
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:145]:
```suggestion
#### Example
```

File: content/ri/using-redisinsight/proxy.md (hunk 2 above)
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:136]:
```suggestion
When `RIPROXYPREFIX` is being set to a value, default being `X-Forwarded-Prefix`, RedisInsight extracts the path from this field in the HTTP Header. So the subpath is actually set by the proxy server and not RedisInsight and this variable just tells which HTTP header field to check for the proxy subpath. Using this approach, multiple proxies can send requests to same RedisInsight with different proxy subpaths.
```

**Review by nermiller — COMMENTED** [review]
@GnaneshKunal I left you a bit of feedback and a couple of questions

**Comment by GnaneshKunal** [comment]
@nermiller Can you re-raise the last suggestion

**Comment by GnaneshKunal** [comment]
and please apply it and merge

File: content/ri/release-notes/v1.13.0.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+description: RedisInsight v1.13.0
+weight: 2
+---
+
+## 1.13.0 (August 2022)
+
+This is the General Availability Release of RedisInsight 1.13 (v1.13.0).
+
+
+## Headlines:
```
Comment (nermiller) [inline content/ri/release-notes/v1.13.0.md:14]:
```suggestion
## Headlines
```

File: content/ri/release-notes/v1.13.0.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+- Subpath Proxy Support
+
+## Details
+
+### Core
+ - Subpath Proxy support: RedisInsight can now be proxied behind a subpath
```
Comment (nermiller) [inline content/ri/release-notes/v1.13.0.md:20]:
```suggestion
- Subpath Proxy support: RedisInsight can now be proxied behind a subpath
```

File: content/ri/using-redisinsight/proxy.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+    depends_on:
+      - redisinsight
+    networks:
+      - redis-network
+
+networks:
+  redis-network:
+    driver: bridge
+```
+
```
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:51]:
```suggestion
```

File: content/ri/using-redisinsight/proxy.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+
+## Subpath proxy
+
+
```
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:55]:
```suggestion
```

File: content/ri/using-redisinsight/proxy.md [hunk 9]
```diff
@@ -0,0 +1,239 @@
...
+     proxy_pass             http://redisinsight:8001/; # Assumes redisinsight runs in this host
+     proxy_read_timeout     900;
+     proxy_set_header       Host $host;
+ }
+}
+```
+
+![RIPROXYPATH](/images/ri/riproxypath.png)
+
+
```
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:131]:
```suggestion
```

File: content/ri/using-redisinsight/proxy.md [hunk 10]
```diff
@@ -0,0 +1,239 @@
...
+When `RIPROXYPREFIX` is being set to a value, default being `X-Forwarded-Prefix`, RedisInsight extracts the path from this field in the HTTP Header. So the subpath is actually set by the proxy server and not RedisInsight and this variable just tells which HTTP header field to check for the proxy subpath. Using this approach, multiple proxies can send requests to same RedisInsight with different proxy subpaths.
+
+{{< note >}}
+
+When the dynamic subpath is used, RedisInsight is reachable both from the normal path, i.e., `localhost:8001` and the one from dynamic subpath.
+
+{{< /note >}}
+
```
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:142]:
```suggestion
```

File: content/ri/using-redisinsight/proxy.md [hunk 11]
```diff
@@ -0,0 +1,239 @@
...
+    depends_on:
+      - redisinsight
+    networks:
+      - redis-network
+
+networks:
+  redis-network:
+    driver: bridge
+```
+
```
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:214]:
```suggestion
```

File: content/ri/using-redisinsight/proxy.md [hunk 12]
```diff
@@ -0,0 +1,239 @@
...
+     proxy_set_header       Host $host;
+ }
+}
+```
+
+![RIPROXYPATH](/images/ri/riproxypath.png)
+
+
+### RIPROXYPREFIX dynamic subpath
+
```
Comment (nermiller) [inline content/ri/using-redisinsight/proxy.md:133]:
```suggestion
```

**Review by nermiller — COMMENTED** [review]
Deleted extra lines and spaces

**Comment by nermiller** [comment]
> @nermiller Can you re-raise the last suggestion

Which one was that? I couldn't parse it out.

**Comment by nermiller** [comment]
@GnaneshKunal let me know if you'd like to push to the latest.

**Comment by GnaneshKunal** [comment]
@nermiller Yes.

**Comment by GnaneshKunal** [comment]
@nermiller Lets merge this

**Comment by nermiller** [comment]
@GnaneshKunal it is live but it's not ordered right. I'll move it up and republish.

**Pull request merged**